gen-tests.gradle.kts

```diff
@@ -99,8 +99,8 @@ val generateGsonAdapterJavaTests by tasks.creating(JavaExec::class) {
   classpath = wire
   main = "com.squareup.wire.WireCompiler"
   args = listOf(
-          "--proto_path=wire-library/wire-tests/src/commonTest/shared/proto",
           "--proto_path=wire-library/wire-tests/src/commonTest/shared/proto/proto2",
+          "--proto_path=wire-library/wire-tests/src/commonTest/shared/proto/proto3",
```

Comment:
This caused some files to be double-included, which had the unfortunate side-effect of triggering our enums warning. Gotta fix that too!

https://github.com/square/wire/issues/2124